open_lm/data.py

```diff
@@ -36,6 +39,12 @@
 from webdataset.mix import RandomMix
 
 
+def seed_worker(worker_id):
+    worker_seed = torch.initial_seed() % 2**32
```

Comment:
Do we need to also set the torch seed here? Also, this doesn't use worker_id, why are we taking it as a parameter?

Reply:
Worker_id is taken as a parameter to satisfy the signature of this function, since it is supposed to be called with an `int` as input.

The code was taken from here https://pytorch.org/docs/stable/notes/randomness.html#dataloader: the torch seed is already set beforehand to be correct, this is just to satisfy possible weirdness when using other libs.